canonical-web-and-design/canonical.com
# Add redirect from '/landscape/dedicated-edition' to 'ubuntu.com/support'

## Done
- Adds a redirect from /enterprise-services/ubuntu-advantage/landscape/dedicated-edition to ubuntu.com/support

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the redirect works
- Compare against the [chat thread](https://github.com/canonical-web-and-design/canonical.com/issues/433) to make sure it fixes

## Issue / Card

Fixes https://github.com/canonical-web-and-design/canonical.com/issues/433
